docs/how_to_guides/keeping_passkeys_in_sync.rst

```diff
@@ -0,0 +1,140 @@
+.. _keeping_passkeys_in_sync:
+
+Keeping user details in sync with Passkeys
+==========================================
```

Comment:
Let's use imperative verbs and also make the heading consistent with the title of the page:

```suggestion
.. _keep_passkeys_up_to_date:

Keep passkeys up to date with user details
==========================================
```